src/silo/common/panic.test.cpp

```diff
@@ -87,12 +99,14 @@ TEST(panic, debugAssertGeWorks) {
 
 #if DEBUG_ASSERTIONS
 
+   const char* old_env = getenv("SILO_PANIC");
    setenv("SILO_PANIC", "", 1);
    try {
       DEBUG_ASSERT_GE(1 + 5, 7);
    } catch (const std::exception& ex) {
       assertMsg(ex.what(), "DEBUG_ASSERT_GE failure: 1 + 5 >= 7: 6 >= 7");
    };
```

Comment:
Yes, that would be better. The panics should always either throw or abort though, I'm not that worried that they wouldn't do either, basically the test here is that it doesn't abort (and checks the message at the same time). But it would be better. But that will have to be another PR (it's independent), and low priority, I'll just add a note on my side.